contracts/DependencyRegistryV1.sol

```diff
@@ -0,0 +1,1074 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.17;
+
+// Created By: Art Blocks Inc.
+
+import "./interfaces/0.8.x/IAdminACLV0.sol";
+import "./interfaces/0.8.x/IDependencyRegistryCompatibleV0.sol";
+import "./interfaces/0.8.x/IDependencyRegistryV0.sol";
+
+import "@openzeppelin-4.7/contracts/utils/Strings.sol";
+import "@openzeppelin-4.7/contracts/utils/structs/EnumerableSet.sol";
+import "@openzeppelin-4.8/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin-4.8/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin-4.5/contracts/utils/math/SafeCast.sol";
+
+import {IContractScript} from "scripty.sol/contracts/scripty/IContractScript.sol";
+import {AddressChunks} from "./AddressChunks.sol";
+import {IScriptyBuilder, WrappedScriptRequest} from "scripty.sol/contracts/scripty/IScriptyBuilder.sol";
+
+import "./libs/0.8.x/BytecodeStorage.sol";
+import "./libs/0.8.x/Bytes32Strings.sol";
+
+import "hardhat/console.sol";
```

Comment:
interesting I've never used this 👀  are you using this functionality at all?

Reply:
Oops, didn't mean to leave this in, was just using for debugging